MediaBrowser.Controller/Entities/BaseItem.cs

```diff
@@ -1517,10 +1517,10 @@ public virtual bool IsFolder
 
         public virtual string GetClientTypeName()
         {
-            if (IsFolder && SourceType == SourceType.Channel && !(this is Channel))
-            {
-                return "ChannelFolderItem";
-            }
+            ////if (IsFolder && SourceType == SourceType.Channel && !(this is Channel))
+            ////{
```

Comment:
This is going to negatively impact apps that are looking for ChannelFolderItem in order to apply special considerations. While I would like to remove this, at the moment I don't really want to be dragged into all of the app work that this is going to require.